library/common/config_template.cc

```diff
@@ -54,7 +86,19 @@ const char* config_template = R"(
       cluster_name: base_wwan
       endpoints: *base_endpoints
     transport_socket: *base_transport_socket
+    upstream_connection_options: *upstream_opts
     type: LOGICAL_DNS
+  - name: dynamic_forward_proxy_cluster
```

Comment:
IMO we should use this to replace the base and base_* clusters. It's really more the standard use case.